demisto_sdk/commands/doc_reviewer/doc_reviewer.py

```diff
@@ -150,7 +150,7 @@ def get_files_from_git(self):
             ) in self.SUPPORTED_FILE_TYPES:
                 self.files.append(file)
 
-    def get_files_to_run_on(self, file_path):
+    def get_files_to_run_on(self, file_path=None):
         """Get all the relevant files that the spell-check could work on"""
         if self.git_util:
             self.get_files_from_git()
```

Comment:
seems duplicate to what we have in `get_files_to_run_on()`, consider using the latter instead.